Lib/pdb.py

```diff
@@ -527,6 +536,13 @@ def message(self, msg):
     def error(self, msg):
         print('***', msg, file=self.stdout)
 
+    # convenience variables
+
+    def set_convenience_variable(self, frame, name, value):
+        if '__pdb_convenience_variables' not in frame.f_globals:
```

Comment:
Is there a reason to have two underscores in the key?

Reply:
Not really, that's just an arbitrary choice. It feels more "private" to me compared to single underscore. `pdb` uses `__return__` and `__exception__` in local variables, which are dunderscore (and yes they have trailing underscores as well). Do you think it would be better to use `__pdb_convenience_variables__` so it's more officially "internal"? The real purpose here is just to avoid name conflict as much as possible.

Reply:
> Not really, that's just an arbitrary choice. It feels more "private" to me compared to single underscore. `pdb` uses `__return__` and `__exception__` in local variables, which are dunderscore (and yes they have trailing underscores as well). Do you think it would be better to use `__pdb_convenience_variables__` so it's more officially "internal"? The real purpose here is just to avoid name conflict as much as possible.

Perhaps. I'm not sure how much it matters. Up to you.

Reply:
I think `__pdb_convenience_variables` is fine. The naming conflict possibility is extremely low anyway.